tests/UnifiedSpecTests/UnifiedSpecTest.php

```diff
@@ -60,57 +60,10 @@ class UnifiedSpecTest extends FunctionalTestCase
         'valid-pass/expectedEventsForClient-eventType: eventType can be set to command and cmap' => 'PHPC does not implement CMAP',
         'valid-pass/expectedEventsForClient-eventType: eventType defaults to command if unset' => 'PHPC does not implement CMAP',
         // Change stream "comment" option is not yet implemented
-        'change-streams/change-streams: Test with document comment' => 'Not yet implemented (PHPLIB-749)',
-        'change-streams/change-streams: Test with document comment - pre 4.4' => 'Not yet implemented (PHPLIB-749)',
-        'change-streams/change-streams: Test with string comment' => 'Not yet implemented (PHPLIB-749)',
         'change-streams/change-streams: Test that comment is set on getMore' => 'Not yet implemented (PHPLIB-749)',
         'change-streams/change-streams: Test that comment is not set on getMore - pre 4.4' => 'Not yet implemented (PHPLIB-749)',
```

Comment:
These tests should no longer be skipped. Same reason as https://github.com/mongodb/mongo-php-library/pull/925/files#r880003993.